src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: String }) title = "";
+
+  @property({ type: Boolean }) expanded = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="summary ${this.expanded ? "expanded" : ""}"
+        @click=${this._togglePanel}
+      >
+        <div class="summary-content">
+          <p class="summaty-title">
+            ${this.title}
```

Comment:
I feel like this would be better as a slot `<slot name="title"></slot>` So that we can insert more than just a title

Reply:
![image](https://user-images.githubusercontent.com/18730868/92550220-a63b4980-f220-11ea-9927-b23688f653ff.png)

Reply:
done, there are two slots: `title` and `content`.
Feedback is more than welcome 🙂